tests/test_web.py

```
@@ -910,3 +910,222 @@ def test_export_includes_score_breakdown(self, client, settings):
+        """VIX == 15 classifies as Normal (boundary)."""
+        from option_alpha.web.routes import _classify_regime
+
+        label, hint = _classify_regime(15.0)
+        assert label == "Normal"
+        assert hint == "Standard strategies apply"
+
+    def test_normal_at_19_99(self):
+        """VIX == 19.99 classifies as Normal."""
+        from option_alpha.web.routes import _classify_regime
+
+        label, hint = _classify_regime(19.99)
```

Ruff (0.15.0)

[warning] Unpacked variable `hint` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)